src/pages/Transaction/TransactionComp.tsx

```diff
@@ -327,6 +357,80 @@ const handleCellbaseInputs = (inputs: State.Cell[], outputs: State.Cell[]) => {
+  const { isCellbase } = transaction
+
+  const query = useQuery(['ckb_transaction_details', txHash], async () => {
+    const ckbTransactionDetails = await fetchTransactionLiteDetailsByHash(txHash)
+    return ckbTransactionDetails.data
+  })
+
+  const transactionLiteDetails: State.TransactionLiteDetails[] = query.data ?? defaultTransactionLiteDetails
+
+  return (
+    <>
+      {transactionLiteDetails &&
+        transactionLiteDetails.map((item, index) => {
+          return (
+            <div className="transaction_lite" key={index}>
+              <div className={styles.transactionLiteBox}>
+                <div className={styles.transactionLiteBoxHeader}>
+                  <div className={styles.transactionLiteBoxHeaderAddr}>
+                    <Addr address={item.address} isCellBase={isCellbase} />
+                  </div>
+                </div>
+                <div className={styles.transactionLiteBoxContent}>
+                  {item.transfers.map((items, innerIndex) => {
+                    return (
+                      <div key={innerIndex}>
+                        <div>{items.tokenName}</div>
+                        <div>
+                          {(items.transferType && items.transferType === 'nervos_dao_deposit') ||
+                          items.transferType === 'nervos_dao_withdrawing' ? (
+                            <Tooltip
+                              placement="top"
+                              title={
+                                <div>
+                                  {items.transferType === 'nervos_dao_deposit'
+                                    ? i18n.t('transaction.nervos_dao_deposit')
+                                    : i18n.t('transaction.nervos_dao_withdraw')}
+                                </div>
+                              }
+                            >
+                              <span className={styles.tag}>
+                                {isDaoWithdrawCell(items.transferType)
+                                  ? i18n.t('nervos_dao.withdraw_tooltip')
+                                  : i18n.t('nervos_dao.withdraw_request_tooltip')}
+                              </span>
+                            </Tooltip>
+                          ) : null}
+                          {items.transferType === 'nft_transfer' ? (
+                            <span className={styles.nftId}>-ID : {items.nftId}</span>
+                          ) : (
+                            <>
+                              <span className={items.capacity > 0 ? styles.add : styles.subtraction}>
+                                {items.capacity > 0 ? '+' : ''}
+                              </span>
+                              <DecimalCapacity
+                                color={items.capacity > 0 ? '#00CC9B' : '#FA504F'}
+                                value={localeNumberString(shannonToCkb(items.capacity))}
+                              />
```

Comment:
`Color` and `prefix` can be done by CSS to simplify the component

Ref:
- pseudo element: https://stackoverflow.com/questions/16392412/how-to-prefix-this-input-element-using-css